partner_event/models/event_registration.py

```diff
@@ -5,6 +5,7 @@
 # Copyright 2020 Tecnativa - Víctor Martínez
 # Copyright 2014-2023 Tecnativa - Pedro M. Baeza
 # Copyright 2023 Tecnativa - Carolina Fernandez
+# Copyright 2024 Tecnativa - Juan José Seguí
```

Comment:
To change a tiny bit of the code doesn't entitle authorship. Put yourself in the contributors thou

Reply:
This is not really change authorship, but adding the copyright. It can be accepted if there are enough changes in the file (and not only cosmetic).